lib/enums/task-schedule.enum.ts

```diff
@@ -0,0 +1,89 @@
+export enum TaskSchedule {
```

Comment:
What about the `CronExpression` name instead?

Reply:
Sounds good!